recipe/recipe.yaml

```diff
@@ -0,0 +1,54 @@
+context:
+  name: pymupdf
+  version: 1.24.10
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com/pymupdf/PyMuPDF/archive/${{ version }}.tar.gz
+  sha256: 76fc037d37e6879af5b3507c08f34f737852bf3ce5dc0968aa23b8ef79dddb02
+
+build:
+  number: 0
+  skip:
+    - not linux
+  script: PYMUPDF_SETUP_MUPDF_BUILD="" PYMUPDF_SETUP_MUPDF_REBUILD=0 PYMUPDF_MUPDF_LIB="$PREFIX/lib" PYMUPDF_INCLUDES="$PREFIX/include -I$PREFIX/include/mupdf -I$PREFIX/include/freetype2" pip install . -vv --no-deps --no-build-isolation
+
+requirements:
+  build:
+    - ${{ compiler('c') }}
+    - ${{ compiler('cxx') }}
+    - ${{ stdlib("c") }}
+    - swig
+    - mupdf =1.24.9
+  host:
+    - python
+    - pip
+    - mupdf =1.24.9
```

Comment:
Should not `pymupdf=1.24.10` use `mupdf=1.24.9` ?

Reply:
yes, their [release notes](https://github.com/pymupdf/PyMuPDF/releases/tag/1.24.10) mention that for some reason `1.24.10` uses mupdf `1.24.9`

Reply:
Good catch - I just realised the same also applied for `pymupdf=1.24.9` using `mupdf=1.24.8`. This is quite confusing but I guess it is nothing we can change right now. Can you merge with `main` then I can move ahead and merge this pull request.